src/constants.js

```diff
@@ -1,7 +1,9 @@
 /* @flow */
-
-const userHome = require('user-home');
 const path = require('path');
+let userHome = require('user-home');
+if (process.platform === 'linux' && process.env.USER === 'root') {
+  userHome = path.resolve(process.execPath, '..', '..', 'lib');
```

Comment:
This makes an assumption about where Node.js is installed, which could vary based on the environment (eg. `/usr/bin/node` if the user installed it via a package manager, `/usr/local/bin/node` or `/opt/nodejs/` or really anything else if they compiled it themselves).

If you want to use `/usr/local/lib/` then it's probably best to just hard-code that here.